src/v2/Apps/Order/Components/ArtworkSummaryItem.tsx

```diff
@@ -41,6 +46,12 @@ const ArtworkSummaryItem: React.FC<ArtworkSummaryItemProps> = ({
     textOverflow: "ellipsis",
   } as any
 
+  const artworkPrice = getOfferItemFromOrder(lineItems)
```

Comment:
The "List Price" elsewhere is using the last offer price, so I did the same here, but I'm wondering if this should not be the case. Shouldn't the list price be the first given price regardless of the offer amount changes?